caps/ajio_browserstack_capabilities.json

```diff
@@ -50,7 +50,14 @@
           "cloud": {
             "cloudName": "browserstack",
             "url": "https://hub.browserstack.com",
-            "apiUrl": "https://api-cloud.browserstack.com/app-automate/"
+            "apiUrl": "https://api-cloud.browserstack.com/app-automate/",
+            "devices": [
+              {
```

Comment:
in teswiz, we populate the devices at runtime. Why do we need to add this here?

Reply:
as part of the examples, I added devices info,
have removed it.